code/modules/projectiles/projectile/bullets.dm

```diff
@@ -62,13 +62,15 @@
 	fire_sound = 'sound/weapons/Gunshot_c20.ogg'
 
 /obj/item/projectile/bullet/midbullet/lawgiver
-	damage = 10
+	damage = 15
 	stun = 0
 	weaken = 0
 	superspeed = 1
 
 /obj/item/projectile/bullet/midbullet2
-	damage = 25
+	damage = 20
+	stun = 0
+	weaken = 0
```

Comment:
You know I always had that odd feeling that I should comment more on the changes of a PR because it seems nobody here catches shit like this.

I saw the level 1 laser scalpel(whatever it was, red edge one) cost being reset by ComicIronic's PR but I didn't comment on it because _I thought a coder would "check" the PR like they do for every PR(Thus the long wait times which are definitely used to check the code and not to make everything a mess for everyone)_